tools/nodejs/binding.gyp.in

```diff
@@ -9,15 +9,16 @@
                 "src/connection.cpp",
```

Comment:
Is it the template version of this file that we're shipping? Have we checked that the built tar works as expected?

Reply:
I would suspect the tar build works fine as long as we are copying over the source files, but it will not work when doing an incremental build. Does this happen as part of the `node-pre-gyp package` command? Or is the tar file uploaded elsewhere separately?

Reply:
I tested out `npm pack` followed by `npm install <tarball>` locally and it works - but if the tarball is built and uploaded as part of the CI we need to ensure the tarball is created from a fresh `configure`, and not from a `configure` that is linked to an existing build.